### PR TITLE
D2K - Add Ordos Mission 1a

### PR DESCRIPTION
**Pull request by MustaphaTR**
Lua is mostly copy-paste from Atr1a. Map is imported via utility commands, i just did some editing.

### CODE REVIEW

File: mods/d2k/maps/ordos-01a/map.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		LockColor: True
+		Color: B3EAA5
+		Enemies: Harkonnen, Creeps
+	PlayerReference@Harkonnen:
+		Name: Harkonnen
+		LockFaction: True
+		Faction: harkonnen
+		LockColor: True
+		Color: FE0000
+		Enemies: Ordos
```
Comment (ltem) [inline mods/d2k/maps/ordos-01a/map.yaml]: There is no `Creeps` in the Harkonnen Enemies line, but there is one in the Atreides 01a Mission ([LinkToLine](https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/maps/atreides-01a/map.yaml#L46)). 
I think without it the Harkonnen are more "focused" to fight their enemies, so I prefer it. Still I would like to hear your opinion about it.

**Comment by ltem** [comment]
:+1: no problems during playing

File: mods/d2k/maps/ordos-01a/map.yaml (hunk 1 above)
Comment (MustaphaTR) [inline mods/d2k/maps/ordos-01a/map.yaml]: I removed it intentionally. Stupid AI is trying to attack the worm instead of us otherwise.

**Review by ltem — APPROVED** [review]

**Review by Mailaender — APPROVED** [review]

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/3b0c30789953985b6c0a73af48ce243bb949478b...085ef8911b273a483bff293ed9ec03d566633c2f)

**Pull request merged**